ruby/lib/transformer/syskit/extensions/system_network_generator.rb

```diff
@@ -0,0 +1,71 @@
+module Transformer
+    # Module used to add some functionality to Syskit::NetworkGeneration::Engine
+    module SystemNetworkGeneratorExtension
+        # During network validation, checks that all required frames have been
+        # configured
+        def validate_generated_network
+            super
+
+            if Syskit.conf.transformer_enabled?
+                plan.find_local_tasks(Syskit::TaskContext).each do |task|
+                    next if !(tr = task.model.transformer)
```

Comment:
Did we remove the "prefer unless over if not" cop or is this here just because you copied this code from pre-rubocop days?

Reply:
Second one. Like beginning of flatfish days.

This is one package that needs a rubocop cleanup